odl/contrib/fom/supervised.py

```diff
@@ -201,22 +215,32 @@ def mean_value_difference(data, ground_truth, mask=None, normalized=False,
 
     The normalized variant takes values in :math:`[0, 1]`.
     """
-    l1_norm = odl.solvers.L1Norm(data.space)
+    try:
+        space = data.space
```

Comment:
data = space.element(data)?

Reply:
We need `space` in the first place, no?

Reply:
I guess they could technically be different, no?

Reply:
Isn't `data.space.element(data) is data` always true?